bindings/wasm/examples/examples.js

```diff
@@ -0,0 +1,170 @@
+const examples = new Map();
+
+function appendExample(name, func) {
+  const whole = func.toString();
+  const lines = whole.split('\n');
+  lines.splice(0, 1);// remove first line
+  lines.splice(-2, 2);// remove last two lines
+  for (const line of lines) {
+    line.substring(2);// remove first two leading spaces
+  }
+  const body = '\n' + lines.join('\n');
+  examples.set(name, body);
+}
+
+function Intro() {
+  // Write code in JavaScript or TypeScript and this editor will show the API docs.
+  // Manifold constructors include "cube", "cylinder", "sphere", "extrude", "revolve".
+  // Type e.g. "box." to see the Manifold API.
+  // Use console.log() to print output (lower-right).
+  // This editor defines Z as up and units of mm.
+  const box = cube([100, 100, 100], true);
+  const ball = sphere(60, 100);
+  // You must name your final output "result".
+  const result = box.subtract(ball);
+  return result;
+}
+appendExample('Intro', Intro);
```

Comment:
I think that the normal way to do this would be to put all these functions as object fields, and iterate over those fields? I guess that would be a bit cleaner. To add spaces, we can just require these function names to be in camel case and split them when the case changes from lower case to upper case.

Reply:
Good call, I'll do that.